prow/config.yaml

```diff
@@ -396,6 +396,67 @@ tide:
     - do-not-merge/work-in-progress
     - needs-ok-to-test
     - needs-rebase
+  - repos:
+    - kubernetes/kubernetes
+    milestone: v1.12
+    includedBranches:
+    - master
+    - release-1.12
+    labels:
+    - lgtm
+    - approved
+    - priority/critical-urgent
+    - "cncf-cla: yes"
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/blocked-paths
+    - do-not-merge/cherry-pick-not-approved
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/release-note-label-needed
+    - do-not-merge/work-in-progress
+    - needs-rebase
+  - repos:
+    - kubernetes/kubernetes
+    excludedBranches:
+    - master
+    - release-1.12
+    labels:
+    - lgtm
+    - approved
+    - "cncf-cla: yes"
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/blocked-paths
+    - do-not-merge/cherry-pick-not-approved
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/release-note-label-needed
+    - do-not-merge/work-in-progress
+    - needs-rebase
+  - repos:
+    - kubernetes/kubectl
+    labels:
+    - "cncf-cla: yes"
+    missingLabels:
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    reviewApprovedRequired: true
+  - repos:
+    - kubernetes-sigs/federation-v2
+    - kubernetes-sigs/kubeadm-dind-cluster
+    - kubernetes-sigs/testing_frameworks
+    - kubernetes-sigs/poseidon
```

Comment:
Are these kubernetes-sigs repos supposed to be here? They look like they're covered by the large query above.

Reply:
Yeah, this looks like a revert of https://github.com/kubernetes/test-infra/pull/9224. Perhaps got caught up in the rebase?

Reply:
Should be fixed now.